tests/zkevm/test_worst_compute.py

```diff
@@ -1703,3 +1703,47 @@ def test_worst_calldataload(
         post={},
         tx=tx,
     )
+
+
+@pytest.mark.parametrize(
+    "opcode,topic_count",
```

Comment:
Maybe we can remove the `topic_count` parameter, and derive it from `opcode - Op.LOG0`?
Mainly because combinations like `Op.LOG0, 1` or `Op.LOG3, 4` would never make sense?

Reply:
I like this suggestion :smile: